rasa/core/policies/ensemble.py

```diff
@@ -604,18 +545,12 @@ def _pick_best_policy(
                 continue
 
             confidence = (prediction.max_confidence, prediction.policy_priority)
-            if self._is_form_policy(policy_name):
-                # store form prediction separately
-                form_confidence = confidence
-                form_policy_name = policy_name
-            elif confidence > best_confidence:
+            if confidence > best_confidence:
                 # pick the best policy
                 best_confidence = confidence
                 best_policy_name = policy_name
 
-        if form_confidence is not None and self._is_not_mapping_policy(
-            best_policy_name, best_confidence[0]
-        ):
+        if form_confidence is not None:
```

Comment:
Is this ever possible now?

Reply:
Great point! It can't happen now. Removing it.